### PR TITLE
[15.0][MIG] account_voucher_killer: Migration to v15

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT48153

@pedrobaeza @victoralmau please review

### CODE REVIEW

File: account_voucher_killer/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2013 Camptocamp SA (http://www.camptocamp.com)
+# @author Nicolas Bessi
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Accounting Payment Access",
+    "summary": "Prevent the usage of payments from invoices",
+    "version": "15.0.1.0.0",
+    "category": "other",
```
Comment (victoralmau) [inline account_voucher_killer/__manifest__.py]:
```suggestion
    "category": "Accounting & Finance",
```

File: account_voucher_killer/security/invoice_data.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="invoice_payment_user" model="res.groups">
+        <field name="name">Payments for Invoices</field>
+        <field name="category_id" ref="base.module_category_hidden" />
```
Comment (victoralmau) [inline account_voucher_killer/security/invoice_data.xml]:
```suggestion
        <field name="category_id" ref="base.module_category_hidden" />
        <field
            name="users"
            eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"
        />
```

**Review by victoralmau — APPROVED** [review]
Functional review OK

File: account_voucher_killer/security/invoice_data.xml (hunk 2 above)
Comment (pilarvargas-tecnativa) [reply account_voucher_killer/security/invoice_data.xml]: done! thanks :smiley:

**Review by pedrobaeza — COMMENTED** [review]
- Please squash administrative commits and some repetitive commits that can be joined from the same author.
- Put the res.groups in "Extra permissions" group instead of "Technical features".

**Comment by pilarvargas-tecnativa** [comment]
> * Please squash administrative commits and some repetitive commits that can be joined from the same author.
> * Put the res.groups in "Extra permissions" group instead of "Technical features".

done!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**